src/ferc_xbrl_extractor/arelle_interface.py

```diff
@@ -28,3 +28,17 @@ def load_taxonomy(path: str):
     view.view(XbrlConst.parentChild, None, None, None)
 
     return taxonomy, view
+
+
+def load_taxonomy_from_archive(filepath: str, archive_path: str):
+    """Load an XBRL taxonomy from a zipfile archive.
+
+    Args:
+        filepath: Path to zipfile on disc.
+        archive_path: Relative path to taxonomy entry point within archive.
+    """
+    # Create arelle FileSource object
+    print(f"archive_path={archive_path}")
```

Comment:
Can this be replaced with a logger method?

Reply:
Oh yeah I'd just put this in for debugging, so I just removed it